composer.json

```diff
@@ -34,6 +34,8 @@
     },
     "require-dev": {
         "mongodb/builder": "^0.2",
+        "league/flysystem-gridfs": "^3.28",
+        "league/flysystem-read-only": "^3.0",
```

Comment:
Should one or both of these go into the `suggest` list?

Reply:
Added `league/flysystem-gridfs` to the suggestions.